lib/resources/r4/related_person.yaml

```diff
@@ -43,14 +43,11 @@ fields:
     action: terminology
     url: http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.communication.language
     binding:
-      description: The language used to communicate with a related person about a patient's health. 
+      description: The language used to communicate with a related person about a patient's health.
       terminology:
       - display: CommonLanguages
```

Comment:
maybe this is supposed to be "Common Languages" ?

Reply:
This is currently consistent with how it is shown on the Patient page. Do you still want to change it?

Reply:
i think it's fine as-is